.github/workflows/push.yml

```diff
@@ -23,33 +23,21 @@ jobs:
   publish:
     needs: [pack, test]
     runs-on: ubuntu-latest
+
     steps:
     - name: Setup .NET
       uses: actions/setup-dotnet@v3
       with:
         dotnet-version: ${{ env.DOTNET_VERSION }}
         source-url: ${{ env.AZURE_ARTIFACTS_FEED_URL }}
       env:
-        NUGET_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }} 
+        NUGET_AUTH_TOKEN: ${MYGET_TOKEN}
+
     - name: Download Express
       uses: actions/download-artifact@v3
       with:
-        name: express
-        path: ./out
-    - name: Download Trace
-      uses: actions/download-artifact@v3
-      with:
-        name: trace
-        path: ./out
-    - name: Download WorkNet
-      uses: actions/download-artifact@v3
-      with:
-        name: worknet
-        path: ./out
-    - name: Download bctklib
-      uses: actions/download-artifact@v3
-      with:
-        name: bctklib
+        name: packages
         path: ./out
+
     - name: Publish to Azure Artifacts
-      run: dotnet nuget push out/*.nupkg
+      run: dotnet nuget push ./out/*.nupkg --source https://www.myget.org/F/neo/api/v2/package --api-key ${MYGET_TOKEN} --symbol-source https://www.myget.org/F/neo/symbols/api/v2/package --symbol-api-key ${MYGET_TOKEN}
```

Comment:
Shouldn't the URL be `https://www.myget.org/F/neo/api/v3/index.json` for the source and i get 404 for URL `https://www.myget.org/F/neo/symbols/api/v2/package`